app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,55 @@
+class TasksController < ApplicationController
+  def index
+    @task_list = Task.order(id: :asc)
```

Comment:
Love how you ordered this! Makes for a better user experience! If the task list were to be used often, you could even order by date_updated (which is an automatic column). That way if a user were to edit an old task it would then be near the top!